### DISCUSSION TITLE
onthisday/10/10-27/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#24] [General]
# onthisday/10/10-27/

Details for On This Day are genarated by the posts added to the website so the content is subject to changes/updates over time.

https://fanzoflenazavaroni.github.io/onthisday/10/10-27/

**Comment by Cjones85** [comment]
Hello Lena! Its been a while since I have written to you. You have been gone 24 years now. The world is worse for your loss. The world is sadly lacking in beautitul sweet kind souls like you. From one tormented soul to another, I sincerely hope you found in death the peace that you never did in life. Love, from a kindred spirit.
